src/SUMMARY.md

```diff
@@ -47,7 +47,8 @@
 
 - [Welcome](welcome-day-1-afternoon.md)
 - [Tuples and Arrays](tuples-and-arrays.md)
-  - [Tuples and Arrays](tuples-and-arrays/tuples-and-arrays.md)
```

Comment:
When deleting a page, you have to add an entry to the redirection table in `book.toml`. That way we avoid 404 pages when people visit the old URL.

In this case, you should redirect `tuples-and-arrays/tuples-and-arrays.html` to `tuples-and-arrays/tuples.html`:

```toml
[output.html.redirect]
"tuples-and-arrays/tuples-and-arrays.html" = "tuples.html"
```

The tricky part is that the path to the right of `=` is relative to the path on the left.

Reply:
Thanks, Noted, I will check and make it works.

Reply:
Thanks!